pkg/cmd/pipeline/start.go

```diff
@@ -395,6 +395,11 @@ func (opt *startOptions) startPipeline(pName string) error {
 		if err != nil {
 			return err
 		}
+		if len(prLast.ObjectMeta.GenerateName) > 0 {
+			pr.ObjectMeta.GenerateName = prLast.ObjectMeta.GenerateName
+		} else {
```

Comment:
Wondering about this else case. Is it needed? Why not use what is [already specified for pipelinerun](https://github.com/tektoncd/cli/blob/0dcf77c35bed4c993064aa993039f4d9e5b9b4f7/pkg/cmd/pipeline/start.go#L381)?

Reply:
Ah, I see the difference for pipelinerun versus pipeline name. Never mind.

Reply:
No problem thank you for your review :smile: